src/language-js/printer-estree.js

```diff
@@ -4154,15 +4181,13 @@ function printFunctionParams(path, print, options, expandArg, printTypeParams) {
   //   b,
   //   c
   // }) {}
-  if (shouldHugArguments(fun)) {
-    return concat([typeParams, "(", join(", ", printed), ")"]);
+  if (isParametersShouldBeHugged) {
```

Comment:
Not sure about this.
Example:
```js
// Input
call((
  one,

  two,
  three,
  four,


  five,
  six,
  seven,
  eight,
  nine,
  ten,

  eleven

) => {});

// Output
call((one, two, three, four, five, six, seven, eight, nine, ten, eleven) => {});
```

Should we respect newlines here?

Reply:
If they can fit in one line.. I would say no.